addons/purchase/static/src/js/purchase_dashboard.js

```diff
@@ -26,10 +26,8 @@ var QWeb = core.qweb;
 // Add mock of method 'retrieve_dashboard' in SampleServer, so that we can have
 // the sample data in empty purchase kanban and list view
 let dashboardValues;
-SampleServer.mockRegistry.add('purchase.order', {
```

Comment:
[REF] web,purchase: SampleServer: mock registry

This commit changes the API of the SampleServer's mock
registry s.t. one can register a mocked version of a method,
whatever the model. The former API required a specific
model to be specified, and only applied on this model.

However, in order to enable the sample data feature on the
cohort view, the 'get_cohort_data' method needs to be
mocked, whatever the concerned model.

One can now register mocked method for a specific
model only (with key 'model/method') or for every
models (with key 'method').

Part of task 2282196

Reply:
done